### DISCUSSION TITLE
How to sync folders?

### DISCUSSION THREAD

**Discussion by br4yd** [vrtmrz/obsidian-livesync#538] [Q&A]
Hello, I have some empty folders in my structure that however are important for the vault structure. They are all on my desktop. However, they only appear on my phone, after I create a file there. They then stay there when empty, because I enabled the setting for this. But when I delete the folder on my computer it doesn't get deleted on my phone.

How can I achieve that my folders are in sync and not only the files itself? The current behavior makes it, so that my whole structure is fine on my computer, but absolutely chaotic on my phone.

Thank you in advance.

**Comment by spcano01** [comment]
Make sure under Wizard, "Enable edge case treatment features" is checked.  Then you will see Patches (blue bandaid icon), click that, and make sure "Keep empty folder is unchecked"?
